src/service/discovery.js

```diff
@@ -165,6 +165,7 @@ const fetchInitialDiscoveryTopics = async (limit = 25, page = 0) => {
  * @property {String} profile_pic_asset_id
  * @property {String} bio
  * @property {String} common
+ * @property {Object} community_info
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `community_info` property is not reflected in the function that uses this typedef. Please ensure that the functions using this typedef are updated to handle this new property.
```diff
-  * @property {Object} community_info
+  // Ensure to update the functions using this typedef to handle 'community_info'
+  * @property {Object} community_info 
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->